fava/help/features.md

```diff
@@ -56,9 +56,13 @@ number or a deep hierarchy of accounts, Fava offers the following options:
 -   `show-closed-accounts`
 -   `show-accounts-with-zero-balance`
 -   `show-accounts-with-zero-transactions`
+-   `collapse-below-level`
 
-Additionally, accounts that have the metadata `fava-collapse-account` set to
-`TRUE` on their Open directive will be collapsed in the account trees.
+The last option will collapse all accounts at or below a given level in the
```

Comment:
Don't think we need to be so verbose here, this detailed explanation rather belongs in `options.md`

Reply:
I have moved the explanation to `options.md`.